icx-proxy/Cargo.toml

```diff
@@ -18,14 +18,13 @@ path = "src/main.rs"
 
 [dependencies]
 anyhow = "1.0.34"
-candid = "0.7"
+candid = "0.7.1"
```

Comment:
Does leaving it with the minor version allow cargo to automatically adopt future patch updates, or was this necessary?

Reply:
It does, but since this is for an executable, including the minor version is appropriate.

Reply:
We need this to ensure `candid` version `0.7.1` or higher in order to ensure the new `ic-types` is used. Yes, a cargo update would automagically use `0.7.1`, but this change is to force the semantic requirement that we need this patch version or greater (i.e. to prevent downstream packages from even being able to shoot themselves in the foot with `candid = "=0.7.0"`).